project-base/scripts/install.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
-set -e
 numberRegex="^[0-9]+([.][0-9]+)?$"
 operatingSystem=""
 allowedValues=(1 2 3)
 projectPathPrefix=""
 echo This is installation script that will install demo Shopsys Framework application on docker with all required containers and with demo database created.
 
+docker ps -q > /dev/null 2> /dev/null
```

Comment:
does `&>` work also for osX shell?

Reply:
will this cover also `docker-compose` binary ? because only docker-compose and docker-sync are used in this script.

Reply:
Yes, &> works in osx. But I'm not sure about windows.
docker-compose and docker-sync just calls Docker if I'm not mistaken, so I believe it should cover all situations.